src/main/java/uk/gov/hmcts/et/common/model/ccd/types/RepresentedTypeR.java

```diff
@@ -15,7 +15,10 @@
 @NoArgsConstructor
 @AllArgsConstructor
 public class RepresentedTypeR {
-
+    @JsonProperty("id")
```

Comment:
I wonder, do we need respondentId? There's respRepName field there for respondent? Also, these fields need to be added to configs?

Reply:
@GauravChawlaMOJ  We cannot rely on resRepName as this can be changed and then the respondent gets unlinked from their rep. Also I have made relevant config changes. PR to follow

Reply:
As discussed, id field is not needed and it can be removed. Also when respondent name is changed , it must be changed in repCollection as well for data to be consistent (even if we are adding respondentId)